src/gui/options_imp.cpp

```diff
@@ -583,6 +587,11 @@ int options_imp::getProxyType() const
     }
 }
 
+int options_imp::limitStep(int value) {
```

Comment:
* coding style, dedicate a new line for `{`
* move `int pow10[7] = {1, 1, 10, 100, 1000, 10000, 100000};` to here.

 add `const` keyword to make it unmodifiable.
 remove `7`, let the compiler count the elements (at compile-time)
 ```c++
 const int pow10[] = {1, 1, 10, 100, 1000, 10000, 100000};
 ```

That's all from me.